thunder/core/jit_ext.py

```diff
@@ -908,12 +908,16 @@ def is_from_torch(fn):
             # Torch functions have __name__ defined
             fn_name = f"{fn.__module__}.{fn.__name__}"
 
-            # For now, only torch-like opaque functions are sharp edges
-            return _general_jit_sharp_edge(
-                f"Trying to call function {fn_name}, but it's unsupported. Please file an issue requesting support.",
-                None,
+            # Probably merge with sharp edges
+            get_calling_opaque_torch_msg = (
+                lambda: f"Trying to call function {fn_name}, but it is not yet supported. "
+                "Please file an issue requesting support. "
+                "For getting a full list of unsupported functions we recommend "
+                "running `examine` from the `thunder.examine` module on your callable/PyTorch module."
```

Comment:
The current wording makes it sound like it will show all unsupported functions in the world

```suggestion
                "For getting a full list of unsupported functions in your function/module we recommend "
                "running `examine` from `thunder.examine`"
```

Reply:
Am I the only one who always gets confused about what the work `module` mean?

Reply:
In PyTorch? All of us. Each person has their own little convention.

My own is "module" when referring to code and implementations and "model" when talking about a specific instance, kind of like `model = MyModule()`. PyTorch Lightning also generally follows this.

Reply:
Could you please elaborate on the wording? Or could you please provide an alternative that you believe is clearer?

Reply:
I mean the confusion between a Python module and a PyTorch `nn.Module`.

Reply:
Maybe let's just skip the word entirely and say

```
To find out which pieces failed to `thunder.jit`, we recommend running `examine`:

from thunder.examine import examine
examine(what_you_passed_to_jit)(*inputs)
```